src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -387,7 +387,7 @@ trait EntityComponent {
 
     def getEntityTypesWithCounts(workspaceContext: SlickWorkspaceContext): ReadAction[Map[String, Int]] = {
       filter(_.workspaceId === workspaceContext.workspaceId).groupBy(e => e.entityType).map { case (entityType, entities) =>
-        (entityType, entities.countDistinct)
```

Comment:
Count Distinct misses rows with null columns.  The all_attribute_values column can be null.  This is true for Entities with no attributes created before the all_attribute_values column was added on 9 Jun 2016.  Newer entities and updated entities always get a string value, thanks to `EntityComponent.createAllAttributesString()`

An alternate solution might be to add a liquibase migration to update all Entities with null all_attribute_values to set it to the empty string.